notebooks/text_classification/solutions/text_generation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>I'd remove all the "TODOs" from the solution notebook, and create "Exercises" in the exercise notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Done!</p>